composer.json

```diff
@@ -88,8 +88,8 @@
         "composer/installers": "1.12.0",
         "cweagans/composer-patches": "1.7.3",
         "danskernesdigitalebibliotek/cms-api": "*",
-        "danskernesdigitalebibliotek/dpl-design-system": "2024.20.1",
-        "danskernesdigitalebibliotek/dpl-react": "2024.20.1",
+        "danskernesdigitalebibliotek/dpl-design-system": "2024.20.2-rc1",
+        "danskernesdigitalebibliotek/dpl-react": "2024.20.2-rc1",
```

Comment:
We have to remember to revert these changes once the related PRs have landed in the respective repos.